setup.cfg

```diff
@@ -3,6 +3,69 @@
 [bdist_rpm]
 doc-files = doc
 
+[options]
+python_requires = >=3.6
+install_requires =
+	joblib>=0.12
```

Comment:
The minimum versions were updated when releasing 0.9 (see readme file). I don't think these updates were taken into account here.

Reply:
Thanks for catching these!